pydm/widgets/baseplot.py

```diff
@@ -101,14 +136,37 @@ def setAxisColor(self, color):
   axisColor = pyqtProperty(QColor, getAxisColor, setAxisColor)
   
   def getPlotTitle(self):
+    if self._title is None:
+      return ""
     return str(self._title)
   
   def setPlotTitle(self, value):
     self._title = str(value)
+    if len(self._title) < 1:
```

Comment:
Initially my thought was that comparing to an empty string literal was maybe slightly slower, but timeit benchmarks prove me wrong.  Not that it matters for something that gets called quite rarely anyway.